Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_PackageShowView.1.html

```diff
@@ -129,7 +129,7 @@ <h4>When working with a Swift Package Manager manifest:</h4>
             <section>
               <ul class="main-metadata">
                 <li class="authors">Written by Author One, Author Two, Author Three, and 5 other contributors.</li>
-                <li class="history">In development for over 5 years, with 
+                <li class="history">In development for 5 years, with 
```

Comment:
This is an interesting diff. The string we injected in the view model `over 5 years` is actually not something our relative date date printing will generate. It only prints `over` for `over 1 year`. For values larger than that it's just "N years".